examples/step-56/doc/intro.dox

```diff
@@ -11,6 +11,8 @@ the programme Melt in the Mantle where work on this tutorial was
 undertaken. This work was supported by EPSRC grant no EP/K032208/1.
 </i>
 
+@dealiiTutorialDOI{https://doi.org/10.5281/zenodo.400995,https://zenodo.org/badge/DOI/10.5281/zenodo.400995.svg}
```

Comment:
Can you move `https://doi.org/` into the macro expansion? This way, you'd only have to actually state the DOI, not the whole link.